install_files/securedrop-app-code/debian/postinst

```diff
@@ -132,10 +132,14 @@ case "$1" in
         if ! grep -qE '^allow-loopback-pinentry$' /var/lib/securedrop/keys/gpg-agent.conf; then
             echo allow-loopback-pinentry >> /var/lib/securedrop/keys/gpg-agent.conf
         fi
+        if ! grep -qE '^default-cache-ttl 0$' /var/lib/securedrop/keys/gpg-agent.conf; then
```

Comment:
Good catch here @nabla-c0d3 - how'd you feel about this being split out into a separate PR in the interests of scope management? AFAICT nothing else in this one would require testing upgrade scenarios, and it would be good to get the bugfix in independently.

Reply:
Sounds good - https://github.com/freedomofpress/securedrop/pull/6174